### DISCUSSION TITLE
训练模型是动态图模型，推理模型是静态图模型。

### DISCUSSION THREAD

**Discussion by reBiocoder** [PaddlePaddle/PaddleOCR#13626] [Q&A]
训练模型是动态图模型，推理模型是静态图模型。

_Originally posted by @GreatV in https://github.com/PaddlePaddle/PaddleOCR/issues/13622#issuecomment-2275608662_

非常感谢！ 请问使用的预训练模型，还是微软的layoutxlm-base吗？ 还是paddle重新训了一个与预训练模型呢？

**Comment by GreatV** [comment]
paddle的
